### ISSUE TITLE
"Fatal error: ID "book.daps.user" has already been defined" on Ubuntu 18.04.2

### ISSUE THREAD

**Issue by cjschroder** [openSUSE/daps#493]
### Problem description
I followed the steps in INSTALL.adoc to install DAPS on Ubuntu 18.04.2 from the tarball, and then from the master branch. Both failed with the same errors. The first failure appears when running `./configure --sysconfdir=/etc` when following the tarball instructions, or at `../bin/daps --dapsroot .. -d DC-daps-user pdf` when installing from GitHub Checkout.

The error is `.build/.profiled/noprofile/daps-asciidoc.xml:213:144: error: element "emphasis" not allowed here`, and the same re 240:84. I corrected the source file, `daps_asciidoc.adoc`, removing the bold tags from `SUBCOMMAND` on line 223, and `DCFILE` on line 196, and then the doc build succeeded.

Then  came the `"Fatal error: ID "book.daps.user" has already been defined"` error. As far as I can tell this is defined correctly, so now I do not know what to do. Any ideas how to fix this, or how to debug it? I will help however I can.

A similar issue here:
https://github.com/openSUSE/daps/issues/430

### Expected behavior

Install without errors.

### Steps to reproduce the behavior

Follow the steps in the Problem Description.

**Comment by fsundermeyer** [comment]
I assume the second error is a follow-up on the first one on daps-asciidoc.xml. That one has been fixed with f3a6d48977c and 2183e0d7ef.

Also note this fix for the installation documentation: dda1e35f591

With these fixes I was able to successfully install https://github.com/openSUSE/daps/archive/develop.zip on Ubuntu 18.02 LTS.

**Comment by cjschroder** [comment]
:+1:  Thank you! I have a successful build \o/. 

I checked my installed Python packages, and was missing python3-lxml, & installed it. I ran `git pull` to get the new commits in Develop, and the docs built fine, but I still got the `"book.daps.user"` error. 

Then I reviewed the required packages list, and was missing Dia. I ran `apt-get install` for all of the required packages, re-ran the configure & make steps, and lo! it did build and now I have DAPS on Ubuntu. Thanks much for fast response!
